ccinput/packages/nwchem.py

```diff
@@ -10,52 +10,127 @@
     get_angle,
     get_dihedral,
     get_npxyz,
+    warn,
+)
+from ccinput.constants import (
+    CalcType,
+    ATOMIC_NUMBER,
+    LOWERCASE_ATOMIC_SYMBOLS,
+    SOFTWARE_MULTIPLICITY,
+    SYN_METHODS
 )
-from ccinput.constants import CalcType, ATOMIC_NUMBER, LOWERCASE_ATOMIC_SYMBOLS
 from ccinput.exceptions import InvalidParameter, ImpossibleCalculation
 
 
 class NWChemCalculation:
-    TEMPLATE = """start {}
+    TEMPLATE = """TITLE "{}"
+    start {}
+    memory total {}
+    charge {}
     geometry units angstroms
+    {}end
+    basis
     {}
     end
-    task {}
-    """
```

Comment:
Extra spaces don't seem problematic in NWchem. In that case, it would be nice to have empty lines to visually separate blocks.

Reply:
Can you push the changes? The pull request will automatically be updated.